For some reason the test fails in Travis CI.

```
=================================== FAILURES ===================================

_________________ TestDataFramePlots.test_mpl2_color_cycle_str _________________

[gw0] linux -- Python 3.7.8 /home/travis/miniconda3/envs/pandas-dev/bin/python

self = <pandas.tests.plotting.test_frame.TestDataFramePlots object at 0x7f022d75ca90>

    def test_mpl2_color_cycle_str(self):

        # GH 15516

        df = DataFrame(randn(10, 3), columns=["a", "b", "c"])

        colors = ["C0", "C1", "C2", "C3", "C4", "C5", "C6", "C7", "C8", "C9"]

        with warnings.catch_warnings(record=True) as w:

            # GH 36972

            warnings.simplefilter("always", "MatplotlibDeprecationWarning")

    

            for color in colors:

                _check_plot_works(df.plot, color=color)

    

            no_warning_raised = len(w) == 0

>           assert no_warning_raised, "MatplotlibDeprecationWarning was raised"

E           AssertionError: MatplotlibDeprecationWarning was raised

E           assert False

pandas/tests/plotting/test_frame.py:183: AssertionError
```

--- a/pandas/tests/plotting/test_frame.py
+++ b/pandas/tests/plotting/test_frame.py
@@ -173,14 +173,18 @@ class TestDataFramePlots(TestPlotBase):
         df = DataFrame(randn(10, 3), columns=["a", "b", "c"])
         colors = ["C0", "C1", "C2", "C3", "C4", "C5", "C6", "C7", "C8", "C9"]
         with warnings.catch_warnings(record=True) as w:
-            # GH 36972
             warnings.simplefilter("always", "MatplotlibDeprecationWarning")
 
             for color in colors:
                 _check_plot_works(df.plot, color=color)
 
-            no_warning_raised = len(w) == 0
-            assert no_warning_raised, "MatplotlibDeprecationWarning was raised"
+            # if warning is raised, check that it is the exact problematic one
+            # GH 36972
+            if w:
+                match = "Support for uppercase single-letter colors is deprecated"
+                warning_message = str(w[0].message)
+                msg = "MatplotlibDeprecationWarning related to CN colors was raised"
+                assert match not in warning_message, msg
 
     def test_color_single_series_list(self):
         # GH 3486

